ethportal-api/src/types/bootnodes.rs

```diff
@@ -2,14 +2,32 @@ use std::str::FromStr;
 
 use anyhow::anyhow;
 
-use crate::types::enr::Enr;
+use crate::types::{enr::Enr, network::Network};
 
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct Bootnode {
     pub enr: Enr,
     pub alias: String,
 }
 
+impl From<Enr> for Bootnode {
+    fn from(enr: Enr) -> Self {
+        let all_bootnodes = DEFAULT_BOOTNODES
```

Comment:
nit: I don't think you have to clone all DEFAULT_BOOTNODES and ANGELFOOD_BOOTNODES. You can iterate over references and clone only if you find match. Something like this:

```rust
Iterator::chain(DEFAULT_BOOTNODES.iter(), ANGELFOOD_BOOTNODES.iter())
    .find(|bootnode| bootnode.enr == enr)
    .cloned()
    .unwrap_or_else(|| Self {
        enr,
        alias: "custom".to_string(),
    })
```